### DISCUSSION TITLE
真实性又该如何验证？或是又一个信息茧房

### DISCUSSION THREAD

**Discussion by theCNONE** [WeNeedHome/SummaryOfLoanSuspension#599] [General]
这里面统计的是真正完完全全的断供，还是只是烂尾楼？
晚上有很多所谓的“烂尾声明”，当然有真的，那是否有假的呢？反正那些“声明”上既没有盖章，也没手印更没签字
我现在发布“重庆保利观澜三期全体业主停贷声明”，是仅仅我停贷，还是真的所有人联合停贷呢？
所谓的停贷里面又有多少“工贼”呢？
在“航空母舰吧”全是兔兔，在“中国房价吧”里面全是神神，在“抗吧”里面全是鼠鼠，真正的因停贷导致的不良资产又占银行资产的多少呢？

**Comment by shijunfeng00** [comment]
那就得看上面愿不愿意赌一把这些都是假的了
